src/LibraryManager/Providers/Unpkg/NpmPackageSearch.cs

```diff
@@ -18,16 +18,19 @@ internal class NpmPackageSearch
 
         public static async Task<IEnumerable<string>> GetPackageNamesAsync(string searchTerm, CancellationToken cancellationToken)
         {
+            if (searchTerm == null)
```

Comment:
String checks should usually use `string.IsNullOrEmpty` unless empty is an allowed value (but in this case, I'm guessing it's not?)

Reply:
hm. I guess it is actually, because below you would use an empty string for the search term.  So this is probably fine

Reply:
Yes. unlike using library name as the key in Editor's npm package completion. In libman it works as a value so after typing colon, we should allow the empty string to invoke completion.